results-mieb/jinaai__jina-clip-v1/1cbe5e8b11ea3728df0b610d5453dfe739804aa9/MSCOCOT2IRetrieval.json

```diff
@@ -0,0 +1,158 @@
+        "nauc_precision_at_1000_std": 0.552726119141797,
+        "nauc_precision_at_100_diff1": 0.15897061095455042,
+        "nauc_precision_at_100_max": 0.42848198127641873,
+        "nauc_precision_at_100_std": 0.24775900337098317,
+        "nauc_precision_at_10_diff1": 0.45241283409781885,
+        "nauc_precision_at_10_max": 0.18182339360260374,
+        "nauc_precision_at_10_std": -0.34048980216449526,
+        "nauc_precision_at_1_diff1": 0.6086677033863201,
+        "nauc_precision_at_1_max": 0.19525350096143335,
+        "nauc_precision_at_1_std": -0.2797335075064764,
+        "nauc_precision_at_20_diff1": 0.42977222993638897,
+        "nauc_precision_at_20_max": 0.20622305981256997,
+        "nauc_precision_at_20_std": -0.31448081933782945,
+        "nauc_precision_at_3_diff1": 0.4972012863500112,
+        "nauc_precision_at_3_max": 0.17750313343860488,
+        "nauc_precision_at_3_std": -0.3218876469389674,
+        "nauc_precision_at_5_diff1": 0.47444239516674885,
+        "nauc_precision_at_5_max": 0.16915063391060323,
+        "nauc_precision_at_5_std": -0.3422618390030155,
+        "nauc_recall_at_1000_diff1": 0.3832069088634286,
+        "nauc_recall_at_1000_max": 0.8901286684038919,
+        "nauc_recall_at_1000_std": 0.8681810401874042,
+        "nauc_recall_at_100_diff1": 0.4185975008775339,
+        "nauc_recall_at_100_max": 0.45763959222653156,
+        "nauc_recall_at_100_std": 0.1007829133427318,
+        "nauc_recall_at_10_diff1": 0.4592974110602662,
+        "nauc_recall_at_10_max": 0.1757829270013686,
+        "nauc_recall_at_10_std": -0.3502055124676758,
+        "nauc_recall_at_1_diff1": 0.6091183832361869,
+        "nauc_recall_at_1_max": 0.19445927993458345,
+        "nauc_recall_at_1_std": -0.2806441521864675,
+        "nauc_recall_at_20_diff1": 0.4522393445441416,
+        "nauc_recall_at_20_max": 0.19461709532525517,
+        "nauc_recall_at_20_std": -0.3418318635602553,
+        "nauc_recall_at_3_diff1": 0.49905878884342847,
+        "nauc_recall_at_3_max": 0.17580402067909676,
+        "nauc_recall_at_3_std": -0.32419684954654177,
+        "nauc_recall_at_5_diff1": 0.4770455272089403,
+        "nauc_recall_at_5_max": 0.1660363394348069,
+        "nauc_recall_at_5_std": -0.3461249105285916,
+        "ndcg_at_1": 0.41602,
+        "ndcg_at_10": 0.58366,
+        "ndcg_at_100": 0.62879,
+        "ndcg_at_1000": 0.63154,
+        "ndcg_at_20": 0.6053,
+        "ndcg_at_3": 0.51938,
+        "ndcg_at_5": 0.55166,
+        "precision_at_1": 0.41602,
+        "precision_at_10": 0.0772,
+        "precision_at_100": 0.00985,
+        "precision_at_1000": 0.00101,
+        "precision_at_20": 0.04291,
+        "precision_at_3": 0.1981,
+        "precision_at_5": 0.1346,
+        "recall_at_1": 0.41549,
+        "recall_at_10": 0.76983,
+        "recall_at_100": 0.9782,
+        "recall_at_1000": 0.99865,
+        "recall_at_20": 0.85476,
+        "recall_at_3": 0.59323,
+        "recall_at_5": 0.67157
```

Comment:
Matches the paper ✅ 
- MSCOCO Image Retr. R@1: 0.4111
- MSCOCO Image Retr. R@5: 0.6644